docs/get-started/README.md

```diff
@@ -1,4 +1,8 @@
 # Getting Started
+**Jexia** - platform that provides backend as a service solution, with a nice set of tools to cover most of the routine tasks during the development and operation process. 
+![About Jexia](./about.png)
+By using Jexia you can save time by skipping development and integration for basic services, as well as host your application on our cloud and do not spend time for preparing, scale, secure, monitoring your environments.   
```

Comment:
for the list: preparing, scale, secure **"and"** monitoring your environments.

Reply:
fixed